src/test/java/edu/umich/its/spe/SPESummaryTest.java

```diff
@@ -34,81 +34,81 @@ public void tearDown() throws Exception {
 	}
 
 	SPESummary ss = null;
-	
 
-	// One sanity check test of simple string setter getter.  We'll trust lombok 
+
+	// One sanity check test of simple string setter getter.  We'll trust lombok
 	// but will do a little verification.
 	@Test
 	public void checkLastDateFound() {
 		ss.setStoredGradesLastRetrieved("LASTDATE");
 		assertEquals("saved date","LASTDATE",ss.getStoredGradesLastRetrieved());
 	}
-	
-	@Test 
+
+	@Test
 	public void checkEmptyUserList() {
 		List<Pair<String,Boolean>> users = ss.getUsers();
 		assertNotNull("got some list from users",users);
 		assertEquals("empty list",0,users.size());
 	}
-	
+
 	@Test
 	public void addOneUser() {
 		ss.appendUser("user 1",true);
 		assertEquals("list of 1",1,ss.getUsers().size());
 	}
-	
+
 	@Test
 	public void addThreeUsers() {
 		ss.appendUser("user 1",true);
 		ss.appendUser("user 2",false);
 		ss.appendUser("user 3",true);
 		assertEquals("list of 3",3,ss.getUsers().size());
 	}
-	
+
 	@Test
 	public void addThreeUsersFluent() {
 		ss.appendUser("user 1",true).appendUser("user 2",false).appendUser("user 3",true);
 		assertEquals("list of 3",3,ss.getUsers().size());
 	}
-	
-//	@Test
-//	public void checkToStringWithUsers() {
-//		ss.appendUser("user 1",true).appendUser("user 2",true).appendUser("user 3",true);
-//		String tos = ss.toString();
-//		assertNotNull("some string",tos);
-//	}
-	
-//	@Test
-//	public void checkToStringFormatLastDateFoundEmpty() {
-//		ss.appendUser("user 1",true).appendUser("user 2",true).appendUser("user 3",true);
-//		String tos = ss.getStoredGradesLastRetrieved().toString();
-//		assertEquals("what?","",tos);
-//	}
-	
+
+	//	@Test
+	//	public void checkToStringWithUsers() {
```

Comment:
Do we still need those test cases? If not, maybe those could be removed altogether?

Reply:
I'll review the tests next time I update this test file.